components/google_cloud/actions/logging-write-log/logging-write-log.mjs

```diff
@@ -1,14 +1,13 @@
-/* eslint-disable camelcase */
-import google_cloud from "../../google_cloud.app.mjs";
+import googleCloud from "../../google_cloud.app.mjs";
 import logSeverity from "../../utils/logSeverity.mjs";
 
 export default {
   name: "Logging - Write Log",
-  version: "0.0.1",
+  version: "0.0.2",
   key: "google_cloud-logging-write-log",
   description: "Writes log data to the Logging service",
```

Comment:
Could you please add a link to the docs here?

Reply:
done